docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     volumes:
       - pg_data:/var/lib/postgresql/data
     ports:
-      - "5432"
+      - "9432:5432"
```

Comment:
keep database port the same to make containerized development easier

Reply:
Might consider leaving this to the docker_compose.override.yaml to allow people to choose their DB port without changing tracked code, in the effort to expose as few ports as possible by default.